c_emulator/sail_riscv_version.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
+
+#include <string>
+
+struct version_info {
```

Comment:
Probably should be a `namespace` rather than a `struct` since that's how it's being used.